bootstrap.ps1

```diff
@@ -22,16 +22,18 @@ Optionally specify the CMake generator string, e.g. "Visual Studio 15
 
 .PARAMETER EnableDebug
 Enable Debug build.
+Deprecated, use --config Debug when building with cmake instead.
 
 .PARAMETER EnableAssert
 Enable Assertions in compiled code (always on for debug build;
 default off in release).
 
 .PARAMETER EnableReleaseSymbols
 Enable symbols with Release build.
+Deprecated, use --config RelWithDebInfo when building with cmake instead.
 
 .PARAMETER EnableCoverage
-Enable build with code coverage support.
+Unsupported.
```

Comment:
Actually supporting code coverage on multi-config generators is more complicated. We will have to define our supported configs by setting `CMAKE_CONFIGURATION_TYPES=Debug;Release;RelWithDebInfo;Coverage` and `CMAKE_MAP_IMPORTED_CONFIG_Coverage=Debug`, and pass these options to the EP dependencies.

I tried to do the latter by passing these two options to `CMAKE_ARGS` in the `ExternalProject_Add` calls, but couldn't make it work. Building the dependencies was failing because `This project doesn't contain the Configuration and Platform combination of Coverage|X64`, and the `CMakeCache.txt` of the EPs showed that the arguments were not passed.

Unless you or someone else have any idea how to fix it, I prefer to keep it unsupported until the superbuild is gone. You can still use code coverage with single-config generators like Makefiles or Ninja.